pkg/config/squash.go

```diff
@@ -123,64 +141,113 @@ func (s *Squash) getDebuggerPodNamespace() string {
 	return s.Namespace
 }
 
-func (s *Squash) ReportOrConnectToCreatedDebuggerPod(createdPod *v1.Pod) error {
+func (s *Squash) ReportOrConnectToCreatedDebuggerPod() error {
 	if s.Machine {
-		fmt.Printf("pod.name: %v", createdPod.Name)
+		// fmt.Printf("pod.name: %v", createdPod.Name)
 	} else {
-		return s.connectUser(createdPod)
+		return s.connectUser()
 	}
 	return nil
 }
 
-func (s *Squash) connectUser(createdPod *v1.Pod) error {
+// TODO - remove this when V2 api is ready
+func (s *Squash) getIntent() squashv1.Intent {
+	return squashv1.Intent{
+		Debugger: s.Debugger,
+		Pod: &core.ResourceRef{
+			Name:      s.Pod,
+			Namespace: s.Namespace,
+		},
+		ContainerName: s.Container,
+	}
+}
+
+func (s *Squash) connectUser() error {
 	if s.Machine {
 		return nil
 	}
-	// Starting port forward in background.
-	portSpec := fmt.Sprintf("%v:%v", s.LocalPort, sqOpts.DebuggerPort)
-	cmd1 := exec.Command("kubectl", "port-forward", createdPod.ObjectMeta.Name, portSpec, "-n", s.getDebuggerPodNamespace())
-	cmd1.Stdout = os.Stdout
-	cmd1.Stderr = os.Stderr
-	cmd1.Stdin = os.Stdin
-	err := cmd1.Start()
+	debugger := local.GetParticularDebugger(s.Debugger)
+	// Refactor - eventually Intent will be created during config/user entry
+	intent := s.getIntent()
+	daClient, err := utils.GetDebugAttachmentClient(context.Background())
```

Comment:
pass context from previous functions
can be in a follow-up

Reply:
https://github.com/solo-io/squash/issues/87